api/test/json-api.e2e-spec.ts

```diff
@@ -14,6 +27,37 @@ describe('JSON API Specs (e2e)', () => {
 
     app = moduleFixture.createNestApplication();
     await app.init();
+
+    /**
+     * Login User
+     */
+
+    const response = await request(app.getHttpServer())
+      .post('/auth/sign-in')
+      .send({
+        username: E2E_CONFIG.users.basic.aa.username,
+        password: E2E_CONFIG.users.basic.aa.password,
+      })
+      .expect(201);
+
+    jwtToken = response.body.accessToken;
+
+    /**
+     * Create Scenario
+     */
+
+    fakeOrganization = await OrganizationsTestUtils.createOrganization(
+      app,
+      jwtToken,
+      E2E_CONFIG.organizations.valid.minimal(),
+    ).then(async (response) => await Deserializer.deserialize(response));
+
+    fakeProject = await ProjectsTestUtils.createProject(app, jwtToken, {
+      ...E2E_CONFIG.projects.valid.minimalInGivenAdminArea({
+        countryCode: fakeCountry,
+      }),
+      organizationId: fakeOrganization.id,
+    }).then(async (response) => await Deserializer.deserialize(response));
```

Comment:
Would be great to make setups (Assets) clean as much as possible, thus I would recommend moving it to standalone function.

Reply:
I do agree on this as well, but I believe this is needs to be talked about and reach to an agreement. It is not a big deal to refactor it later (I think)

Reply:
yep, what we have is already an improvement above the initial implementation (which is still used in a lot of early tests), which in turn was a great improvement above what we had in two previous projects (no e2e tests, and in the earlier no tests at all 😱) but there are better ways.
I suggest to introduce standalone functions bit by bit for new tests, and if/when we have time, update older tests.